test/config.test.js

```diff
@@ -24,6 +24,22 @@ describe('config', () => {
     expect(toArray(config)).toMatchSnapshot();
   });
 
+  test('toArray: String Function', () => {
```

Comment:
I don't see a test here for handling `thenables`. If `Promises/thenables` aren't supported by this change then the README should be updated in this PR as well.